dtfs-central-api/src/services/portal/facility-amendment.submitPortalFacilityAmendmentToUkef.service.test.ts

```diff
@@ -36,6 +36,8 @@ const mockTfm = {
 const bankId = '1';
 const bankName = 'test bank';
 
+const requestDate = getUnixTime(new Date());
```

Comment:
```suggestion
const requestDate = getUnixTime(now());
```

Please use from libs/common.